unbxdAutosuggest.js

```diff
@@ -1224,10 +1237,29 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, undefined) {
 			if (!this.options.popularProducts.title) {
 				this.options.popularProducts.title = 'autosuggest';
 			}
+			if (!this.options.popularProducts.additionalFields) {
```

Comment:
Instead of `additionalFields` and `popularProductFields`, why couldn't you keep one property `popualrProducts.fields`, If there was an internal discussion, Please tell me insight

Reply:
Only popularproducts.additionalFields can be set by the customer config. popularproductFields default config will be picked by by the title, price, image config (if present) and doctype is mandatory. Then both are merged later on and added to the url